data/coreteam_members.yml

```diff
@@ -105,6 +105,11 @@ members:
       
   - name: Jivesh
     description: |
-      a computer science undergrad who's having fun learning new things.
+      a computer science undergraduate who's having fun learning new things.
```

Comment:
Why you change other people's stuff? What if @jivesh identities as a "undergrad" and by changing this you are marginalising his community and erasing his identity?